finality-aleph/src/metrics.rs

```diff
@@ -71,11 +71,25 @@ pub(crate) enum Checkpoint {
 
 #[derive(Clone)]
 pub struct Metrics<H: Key> {
-    inner: Arc<Mutex<Inner<H>>>,
+    inner: Option<Arc<Mutex<Inner<H>>>>,
 }
 
 impl<H: Key> Metrics<H> {
-    pub fn register(registry: &Registry) -> Result<Self, PrometheusError> {
+    pub fn new(registry: Option<&Registry>) -> Metrics<H> {
+        let inner = match registry {
+            Some(r) => Self::try_register(r)
+                .map_err(|err| warn!("Failed to register Prometheus metrics\n{:?}", err))
+                .ok(),
+            None => {
+                warn!("Failed to register Prometheus metrics: registry not available");
```

Comment:
add target to those logs. `aleph-metrics` to be precise. While you are at that you could extract it to const `const LOG_TARGET: &str = "aleph-metrics";` and replace other usages of raw `target: "aleph-metrics"`in this file :D

Reply:
Done, but now we are using `warn!` in a `service.rs` file, so we either need to make `LOG_TARGET` separately defined in `service.rs`, or make it crate-public in `metrics.rs` file.

Reply:
in service.rs it should be some other target so lets not worry about it here. Gj